gateways/ossm-getting-traffic-into-a-mesh.adoc

```diff
@@ -13,4 +13,12 @@ You can expose gateway proxies to traffic outside a cluster by using either a `L
 include::modules/ossm-about-configuring-a-gateway-to-accept-ingress-traffic.adoc[leveloffset=+1]
 include::modules/ossm-exposing-service-using-istio-gateway-and-virtualservice.adoc[leveloffset=+2]
 include::modules/ossm-about-exposing-services-to-traffic-outside-a-cluster.adoc[leveloffset=+1]
-include::modules/ossm-exposing-a-gateway-to-traffic-outside-the-cluster-using-openshift-routes.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/ossm-exposing-a-gateway-to-traffic-outside-the-cluster-using-openshift-routes.adoc[leveloffset=+2]
+
+.Additional resources
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+
+* xref:../gateways/ossm-about-gateways.adoc#ossm-about-gateway-injection_ossm-about-gateways[About gateway injection]
+
+* https://docs.openshift.com/container-platform/4.17/networking/routes/route-configuration.html#nw-creating-a-route_route-configuration[Creating an HTTP-based Route]
```

Comment:
Should this be an xref? Or a link to the latest OCP docs instead of fixed to version 4.17?